StardewArchipelago/GameModifications/Modded/JunimoShopStockModifier.cs

```diff
@@ -146,33 +145,30 @@ private void GenerateYellowJunimoStock(ShopData shopData)
             AddSummerSeedsToYellowStock(shopData);
             AddFallSeedsToYellowStock(shopData);
             AddSaplingsToShop(shopData);
-            shopData.Items.Add(CreateJunimoSeedItem(QualifiedItemIds.RHUBARB_SEEDS));
-            shopData.Items.Add(CreateJunimoSeedItem(QualifiedItemIds.STARFRUIT_SEEDS));
-            shopData.Items.Add(CreateJunimoSeedItem(QualifiedItemIds.BEET_SEEDS));
+            shopData.Items.Add(CreateJunimoSeedItem(QualifiedItemIds.RHUBARB_SEEDS, spring));
```

Comment:
True, but also, what I meant is "why not put them in the above methods for the seasons?"